kitsune/users/management/commands/cleanup_expired_users.py

```diff
@@ -0,0 +1,27 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from kitsune.users.utils import delete_user_pipeline
+
+
+class Command(BaseCommand):
+    help = "Delete users who haven't logged in for more than settings.USER_EXPIRATION_DAYS days"
+
+    def handle(self, *args, **options):
+        User = get_user_model()
+        expiration_date = timezone.now() - timedelta(days=settings.USER_EXPIRATION_DAYS)
```

Comment:
timezone.now() returns a datetime object. Maybe we should convert this to `timezone.now().date()`

Reply:
No the reason for this nit was the naming of the variable. By converting to date() it just removes the time component which as you said it's not relevant for this kind of functionality. This is just nitpicking so treat it as such